homeassistant/components/sunsynk/sensor.py

```diff
@@ -0,0 +1,163 @@
+"""Support for Sunsynk sensors."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, EntityCategory, UnitOfEnergy, UnitOfPower
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    BATTERY_POWER,
+    BATTERY_SOC,
+    DATA_INVERTER_SN,
+    DOMAIN,
+    GRID_ENERGY_EXPORT_TODAY,
+    GRID_ENERGY_EXPORT_TOTAL,
+    GRID_ENERGY_IMPORT_TODAY,
+    GRID_ENERGY_IMPORT_TOTAL,
+    GRID_POWER,
+    SOLAR_ENERGY_TODAY,
+    SOLAR_ENERGY_TOTAL,
+    SOLAR_POWER,
+    SUNSYNK_COORDINATOR,
+)
+from .coordinator import SunsynkCoordinator
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        name="Grid Power",
```

Comment:
Use translation key and set the names in `strings.json`

Reply:
Done